Game/Source/GDKTestGyms/DeltaSerializeStruct.cpp

```diff
@@ -0,0 +1,63 @@
+#include "DeltaSerializeStruct.h"
+#include "Net/UnrealNetwork.h"
+
+void FTestStructItem::PostReplicatedChange(const struct FTestStructContainer& InArraySerializer)
+{
+  int32 ElementIdx = this - InArraySerializer.Items.GetData();
+  if (Material)
+  {
+    int32 ChangeNum = InArraySerializer.Owner->ElementsHavingChanged.FindOrAdd(ElementIdx, 0);
```

Comment:
Doesn't this need to return a reference if you want to modify it afterwards?

Reply:
That was some old prototype, removed now.